src/pudl/transform/classes.py

```diff
@@ -260,6 +260,10 @@ class RenameColumns(TransformParams):
     """
 
     columns: dict[str, str] = {}
+    # A dictionary of columns to be renamed.
+    not_renamed_columns: list[str] = []
+    # A list of raw columns which are expected not to be renamed. Any other
+    # columns in the raw data which fail to be renamed will raise an error.
```

Comment:
If we make these comments into triple-quoted strings they'll show up in the documentation as docstrings on these attributes, which would be wonderful.